debian/daos-server-tests.install

```diff
@@ -9,6 +10,7 @@ usr/bin/bio_ut
 usr/bin/vea_ut
 usr/bin/vos_tests
 usr/bin/ddb_tests
+usr/bin/ddb_ut
```

Comment:
so long as we are adding new unit tests to the package, why don't we run them?  Add entries to utils/utest.yaml ?

Alternatively, if these are just unit tests, you don't necessarily have to install them.  We really should settle on that.  Some tests we don't install, some we do and I don't think it's all that useful for most of them.  Anyway, the alternative here is to forego install and add to ci/test_files_to_stash.txt

Reply:
> why don't we run them? Add entries to utils/utest.yaml ?

Strange. I do add them to `utils/utest.yaml` with this patch. Please double-check.